src/components/Header.js

```diff
@@ -1,6 +1,11 @@
 /*  Added the new Button component.  The props are passed in for the content of the button and its
 		onClick method.
 */
+/*
+		I have updated the method that gets passed to the button here to make it toggle the view on and off.
+		Makes the UI less confusing when trying to return to the main view as before the user had to
+		click the CryptoTraq logo to go back, and it wasn't clear..
+*/
```

Comment:
I have updated the method that gets passed to the button here to make it toggle the view on and off.
It makes the UI less confusing when trying to return to the main view as before the user had to click the CryptoTraq logo to go back, and it wasn't clear.